### PR TITLE
remove k3d dep and namespace the namespace

### PR DESCRIPTION
**Pull request by dviersel**
Nice to have the separate namespaces. It might be an idea to use a namespace variable as input to the templates. Now the "namespace logic" is spread out over multiple files/places.

### CODE REVIEW

File: maven/docker-gatling/pom-init.xml [hunk 1]
```diff
@@ -56,7 +56,13 @@
     	<systemUnderTest>MyAfterburner</systemUnderTest>
         <version>v1.0.0</version>
         <tags>gatling,spring-boot-kubernetes</tags>
-        <apiKey>${ENV.PERFANA_API_KEY}</apiKey>
+        <apiKey>export PERFANA_CONFIG_EXAMPLES_DIR=$(pwd)/perfana-config-examples
```
Comment (ghost) [inline maven/docker-gatling/pom-init.xml]: what happened to the API key?

File: template/grafana-agent.yaml.hbs
Comment (ghost) [inline template/grafana-agent.yaml.hbs:1]: The url and targets also contain the namespace, need to be updated

**Review by ghost — COMMENTED** [review]
See comments

File: maven/docker-gatling/pom-init.xml (hunk 1 above)
Comment (dviersel) [reply maven/docker-gatling/pom-init.xml]: pasted a snippet by accident; is now reverted

File: template/grafana-agent.yaml.hbs
Comment (dviersel) [reply template/grafana-agent.yaml.hbs:1]: fixed

**Pull request merged**